src/movies.js

```diff
@@ -1,16 +1,179 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+function getAllDirectors(movies) {
+    let dirArr = movies.map((i) => {
+        return i.director
+    })
+    return dirArr
+}
+
+function cleanedArray(dirArr) {
+    let cleanArr
+
+}
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies) {
+    let steveDrama = movies.filter((i) => {
+        if (i.director === "Steven Spielberg" && i.genre.includes("Drama")) {
+            return i
+        }
+    }).length
+    return steveDrama
```

Comment:
good use of .includes, putting both conditions in the same line and even chaining the .length to the filter.
By returning the result of the filter instead of saving that inside a variable you could even make the code shorter ;)